radix-engine/src/model/substates/substate.rs

```diff
@@ -263,12 +356,16 @@ impl Into<GlobalAddressSubstate> for Substate {
 }
 
 pub enum SubstateRef<'a> {
+    AuthZone(&'a AuthZone),
+    Worktop(&'a Worktop),
+    Proof(&'a Proof),
+    Bucket(&'a Bucket),
```

Comment:
Can we have `AuthZone/Worktop/Proof/BucketRuntimeSubstate`?

Reply:
Yup